Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -345,9 +356,10 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
 
             ThrowIfDisposed();
             _clientCaps = @params.capabilities;
-            _traceLogging = _clientCaps?.python?.traceLogging ?? false;
-            Analyzer.EnableDiagnostics = _clientCaps?.python?.liveLinting ?? false;
+            _traceLogging = @params.initializationOptions.traceLogging;
+            _analysisUpdates = @params.initializationOptions.analysisUpdates;
 
+            Analyzer.EnableDiagnostics = _clientCaps?.python?.liveLinting ?? false;
```

Comment:
This property should go away in favour of suppressing the individual messages (probably by default, at least for the ones currently hidden by this flag)

Reply:
Yep. We should discuss messages, error codes, etc.